examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -175,11 +175,26 @@
     "--debug_mode", action="store_true", help="Enables debug mode with more info on data preprocessing and evaluation",
 )
 
+parser.add_argument(
+    "--embedding_dimension",
+    default=768,
+    type=int,
+    help="Embedding size of the model",
+)
+
 args = parser.parse_args()
 
 if args.debug_mode:
     logging.setLevel(10)
 
+if args.task_name == "multiwoz":
+    schema_config = {"MAX_NUM_CAT_SLOT": 9, "MAX_NUM_NONCAT_SLOT": 4, "MAX_NUM_VALUE_PER_CAT_SLOT": 50, "MAX_NUM_INTENT": 1}
+else:
+    schema_config = {"MAX_NUM_CAT_SLOT": 6, "MAX_NUM_NONCAT_SLOT": 12, "MAX_NUM_VALUE_PER_CAT_SLOT": 11, "MAX_NUM_INTENT": 4}
```

Comment:
let's do elif args.task_name == sgd: pull constants from data_utils.py, else: raise ValueError "not supported task"

Reply:
If they pass something different, the arg would give error. Already limited there. But would add it here too.

Reply:
By the way, task_name are different for sgd. multiple values.